TaskList/app/controllers/tasks_controller.rb

```diff
@@ -2,10 +2,53 @@ class TasksController < ApplicationController
 
 	def index
 		@tasks = Task.all
+		@working_tasks = Task.working
+		@completed_tasks = Task.completed
 	end
 
 	def show
 		id = params[:id]
 		@task = Task.find(id)
+		@completed_time = @task.completed_at.strftime("Completed %B %e, %Y.") if !@task.completed_at.nil?
```

Comment:
In general I wouldn't recommend including text such as "Completed" in the string provided to strftime, as the choice to include the date string in a statement with other text should be up to the View logic in the template.